docs/en/data_source/catalog/iceberg/iceberg_catalog.md

```diff
@@ -261,6 +261,11 @@ Description: The type of metastore that you use for your Iceberg cluster. Set th
 Required: Yes
 Description: The URI of the REST service endpoint. Example: `https://api.tabular.io/ws`. 
 
+###### iceberg.catalog.view-endpoints-supported
+
+Required: No
+Description: Enables support for View-related operations. Default: `false`
```

Comment:
```suggestion
Description: When the old REST service does not return endpoints in CatalogConfig, whether to use the view endpoints to support view-related operations. This parameter is used for backwards compatibility with older REST servers. Default: `false`
```

Is this better?

Reply:
I also think what you suggested is better. Thank you. I will reflect on it.